docs/en/enterprise-edition/content-collections/dashboards/create-and-manage-dashboards.adoc

```diff
@@ -35,9 +35,11 @@ If the filters in the Edit Dashboard workflow conflict with the filters selected
 |Provides a visualization of alerts coverage.
 
 |*Alerts by MTTR*
-|Displays alerts by severity and their mean time to resolution. 
+|Displays alerts by severity and their mean time to resolution.
+Note: If an alert is opened and resolved more than once a day, the latest resolution time is used for the MTTR calculation. An individual alert may be counted multiple times, if the same alert is resolved many times over multiple days.
```

Comment:
Leave a space between lines 38 and 39 so that the note appears on the next line.